chromium/browser_type.go

```diff
@@ -123,7 +123,7 @@ func (b *BrowserType) Launch(opts goja.Value) (_ api.Browser, browserProcessID i
 	if logger, err = makeLogger(ctx); err != nil {
 		k6ext.Panic(ctx, "setting up logger: %w", err)
 	}
-	launchOpts := common.NewLaunchOptions(k6ext.OnCloud())
+	launchOpts := common.NewLaunchOptions(k6ext.OnCloud(), false)
```

Comment:
I suggest making this a constant to make it more readable. Maybe with: `LocalBrowserMode`?

Reply:
Agree, let me change it!

Reply:
I changed this in a new commit since it involved some refactor. See [c421358](https://github.com/grafana/xk6-browser/pull/800/commits/c421358a0be874c99dc930ddddefaaa1ee7e12f5).